src/Symfony/Component/Validator/Constraints/BackedEnumValue.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Attribute\HasNamedArguments;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
+
+/**
+ * Validates that a backed enum can be hydrated from a value.
+ *
+ * @author Aurélien Pillevesse <aurelienpillevesse@hotmail.fr>
+ */
+#[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
+class BackedEnumValue extends Constraint
+{
+    public const NO_SUCH_CHOICE_ERROR = '53dcc1b1-a8dd-4813-baa5-b8486ff56447';
```

Comment:
Is choice the correct term here?

Reply:
As enums have "cases", I would personally prefer `NO_SUCH_CASE_ERROR` I think.

Reply:
I switched to `NO_SUCH_VALUE_ERROR` because name of the constraint is `BackedEnumVALUE` and the constraint will be triggered when value submitted by a user does not exist